### PR TITLE
added swarm.fund token and sale contracts

### PR DESCRIPTION
**Pull request by wires**
Dear MetaMask,

Here is a PR for the official token and token sale contracts for http://swarm.fund

Thanks,
Jelle.

### CODE REVIEW

**Comment by flyswatter** [comment]
Hi there, thanks for the submission.

The test suite has failed because the images are not in the `images` folder. Please move them and then reply!

**Comment by wires** [comment]
Thanks, didn't notice, that was a silly mistake...

**Comment by flyswatter** [comment]
Hi @wires, I'm trying to confirm these addresses, but I can't find them reported in public anywhere else. Can you point me to a place that I can use to confirm these are official addresses? I do see you on the github/swarmfund group, so I guess that could be proof enough, but another public source would be nice.

File: contract-map.json [hunk 1]
```diff
@@ -1,4 +1,16 @@
 {
+  "0xc3cB5D41C2f79C2F4f33A5D8a6DE72D0D87b1DB3": {
+    "name": "Swarm",
+    "logo": "swarm-logo-token.svg",
+    "erc20": true,
+    "symbol": "SWM",
+    "decimals": 18
+  },
+  "0x3B13F20CB484A87d4613b0ffe2d934d9c70CcCFd": {
```
Comment (flyswatter) [inline contract-map.json]: I was able to confirm this address, but not the other one.

File: contract-map.json [hunk 2]
```diff
@@ -1,4 +1,16 @@
 {
+  "0xc3cB5D41C2f79C2F4f33A5D8a6DE72D0D87b1DB3": {
```
Comment (flyswatter) [inline contract-map.json]: Can you provide a citation for this address? I will need to close this PR if not.

**Comment by kumavis** [comment]
not touching anything associated with joel

**Pull request closed without merging**